bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/PageListItemImpl.java

```diff
@@ -118,7 +127,7 @@ public Resource getTeaserResource() {
                         overriddenProperties.put(ImageResource.PN_LINK_URL, this.getPath());
                     }
                 }
-                teaserResource = new CoreResourceWrapper(resourceToBeWrapped, delegateResourceType, hiddenProperties, overriddenProperties);
+                teaserResource = new CoreResourceWrapper(listResource, delegateResourceType, hiddenProperties, overriddenProperties);
```

Comment:
Shouldn't it be the `resourceToBeWrapped` instead?

Reply:
No, this needs to be the listResource as we want to get the content police of the image delegate component for the same structure element.